### DISCUSSION TITLE
[Feature] external libraries viewable to all users

### DISCUSSION THREAD

**Discussion by ChocolateLoverRaj** [immich-app/immich#14986] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

My family is going to start using Immich. My family already has a folder with photos called "Photos". I don't want those photos managed by Immich, but I want to add it as a read-only external library. This is currently possible, but it has to be owned by a certain user. It would be better if I could make the external library visible to all users (or a list of manually specified users). Since it's read-only it should be less of a problem.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#12614
